package.json

```diff
@@ -6,28 +6,26 @@
     "type": "git",
     "url": "https://github.com/kenchris/urlpattern-polyfill"
   },
-  "main": "./dist/index.umd.js",
-  "module": "./dist/index.js",
-  "unpkg": "./dist/index.umd.js",
-  "types": "./dist/index.d.ts",
-  "exports": "./dist/index.js",
-  "type": "module",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.js",
+    "types": "./dist/index.d.ts"
+  },
   "files": [
-    "src",
     "dist"
   ],
   "devDependencies": {
-    "ava": "^3.14.0",
-    "esm": "^3.2.25",
-    "microbundle": "^0.13.3",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.35.1"
+    "ava": "3",
+    "esm": "latest",
+    "rimraf": "latest",
+    "tsup": "latest",
+    "typescript": "latest"
   },
   "scripts": {
-    "build": "rimraf dist/ && microbundle",
-    "build:watch": "microbundle watch",
-    "test": "npm run build && ava --fail-fast -s",
-    "quicktest": "ava --fail-fast -s",
+    "prebuild": "rimraf dist",
+    "build": "tsup src/index.ts --format esm,cjs --dts",
```

Comment:
This is combining setting up GitHub actions with changing the bundler. Is there a reason to do those together?

Reply:
an again, you should to review this changes at https://github.com/kenchris/urlpattern-polyfill/pull/69

Reply:
But are you fine with these changes @justinfagnani ?

Reply:
I don't know tsup, so 🤷‍♂️ :)